### PR TITLE
Magento_CurrencySymbol: avoid using deprecated escape* methods from A…

### PR DESCRIPTION
**Pull request by sergiynezbritskiy**
### Description (*)

Avoid using deprecated escape* methods from AbstractBlock

### Related Pull Requests

https://github.com/magento/magento2/pull/31610

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
@magento run all tests

File: app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rate/services.phtml [hunk 1]
```diff
@@ -7,10 +7,11 @@
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rate/services.phtml]: please remove not needed closing and opening PHP tags

File: app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rates.phtml [hunk 2]
```diff
@@ -7,10 +7,11 @@
 <?php
```
Comment (ihor-sviziev) [inline app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rates.phtml]: please remove not needed closing and opening PHP tags

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

File: app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rates.phtml (hunk 2 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rates.phtml]: fixed

File: app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rate/services.phtml (hunk 1 above)
Comment (sergiynezbritskiy) [reply app/code/Magento/CurrencySymbol/view/adminhtml/templates/system/currency/rate/services.phtml]: fixed

**Comment by ihor-sviziev** [comment]
@magento run all tests

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, WebAPI Tests

**Review by ihor-sviziev — APPROVED** [review]
✔ Approved.

Failing tests looks not related to changes form this PR.

**Comment by ihor-sviziev** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE

**Pull request closed without merging**